core/network/src/test/sync.rs

```diff
@@ -53,11 +52,6 @@ fn sync_long_chain_works() {
 	let mut net = TestNet::new(2);
 	net.peer(1).push_blocks(500, false);
 	net.sync();
-	// Wait for peer 0 to import blocks received over the network.
```

Comment:
Why were these removed? (I'm not familiar with this test in particular)

Reply:
I think it's a result of this change: https://github.com/paritytech/substrate/pull/1327/files/8113ac554afb9ed430b565e4569d9bc695c06a55#diff-d19df74b5b495aae3892f8f92a5f174aR213

It appears unnecessary to have the extra wait in the test...